libs/isograph-react/src/core/read.ts

```diff
@@ -914,6 +859,122 @@ export function readLinkedFieldData(
   return data;
 }
 
+function isClientPointer(
+  field: ReaderLinkedField,
+): field is ReaderClientPointer {
+  return field.refetchQueryIndex !== null;
+}
+
+export function readClientPointerData(
+  environment: IsographEnvironment,
+  field: ReaderClientPointer,
+  root: StoreLink,
+  variables: Variables,
+  nestedRefetchQueries: RefetchQueryNormalizationArtifactWrapper[],
+  readData: <TReadFromStore>(
+    ast: ReaderAst<TReadFromStore>,
+    root: StoreLink,
+  ) => ReadDataResult<object>,
+): ReadDataResult<unknown> {
+  const refetchReaderParams = readData(
+    [
+      {
```

Comment:
should this be part of the reader AST somehow? Either of the parent reader AST (where the client pointer was selected) or of the client pointer's reader AST?